Android/app/src/main/java/com/macmanus/jamie/bikerentalapp/web/Webservice.java

```diff
@@ -37,4 +38,8 @@
 
     @GET("https://api.github.com/users/{username}")
     Call<GithubUser> getGithubUserData(@Path("username") String username);
+
+    @PUT(" http://34.254.254.125:1234/bike/{id}")
+    Call<User> setBikeStatus(@Path("id") int id, @Query("status") int statusid);
```

Comment:
If you look at the RetrofitInstance.java class you'll see that we have our base URL defined there. So we don't actually have to define the full route in this class. Not necessarily important for this PR but the next time you're editing the webservice maybe make that change.